### PR TITLE
Remove Plaid from tests

### PR DESCRIPTION
**Pull request by tmyracle**
This has ended up being more involved than I would have liked. Spent a good chunk of time trying to figure out why a test wasn't passing and finally realized there was a database function that was doing some calculations that wasn't apparent from the codebase 😑 

Not ready yet, got 2 more tests to replace Plaid in but wanted to show some progress for @MichaelDeBoey 😄

### CODE REVIEW

**Review by MichaelDeBoey — CHANGES_REQUESTED** [review]

File: apps/server/src/app/__tests__/balance-sync.integration.spec.ts
Comment (tmyracle) [reply apps/server/src/app/__tests__/balance-sync.integration.spec.ts:2]: auto import got me 😄

File: prisma/schema.prisma
Comment (tmyracle) [reply prisma/schema.prisma:253]: ok will leave in then, then we can go back and update the tests after your changes land

**Comment by tmyracle** [comment]
@MichaelDeBoey ok made all those changes

File: apps/server/src/app/__tests__/balance-sync.integration.spec.ts
Comment (MichaelDeBoey) [inline apps/server/src/app/__tests__/balance-sync.integration.spec.ts]: My plan was to have both `plaidType` & `category` here

File: apps/server/src/app/__tests__/insights.integration.spec.ts
Comment (MichaelDeBoey) [inline apps/server/src/app/__tests__/insights.integration.spec.ts]: Same here: `plaidType`, `plaidSubtype` **and** `category`

File: apps/server/src/app/__tests__/utils/account.ts
Comment (MichaelDeBoey) [inline apps/server/src/app/__tests__/utils/account.ts]: Same here: `plaidType`, `plaidSubtype` **and** `category`

File: libs/server/features/src/account-balance/investment-transaction-balance-sync.strategy.ts
Comment (MichaelDeBoey) [inline libs/server/features/src/account-balance/investment-transaction-balance-sync.strategy.ts]:
```suggestion
                    it.category IN ('buy', 'sell', 'transfer')
```

File: libs/server/features/src/account/account-query.service.ts
Comment (MichaelDeBoey) [inline libs/server/features/src/account/account-query.service.ts]:
```suggestion
                  it.category = 'transfer'
```

File: libs/server/features/src/account/insight.service.ts
Comment (MichaelDeBoey) [inline libs/server/features/src/account/insight.service.ts]:
```suggestion
                        },
                        {	
                            category: 'dividend',	
                        },
```

File: libs/server/features/src/account/insight.service.ts
Comment (MichaelDeBoey) [inline libs/server/features/src/account/insight.service.ts]:
```suggestion
                it.category = 'transfer'
```

File: apps/server/src/app/__tests__/connection.integration.spec.ts
Comment (MichaelDeBoey) [inline apps/server/src/app/__tests__/connection.integration.spec.ts:8]:
```suggestion
import type { Prisma, AccountConnection, User } from '@prisma/client'
import { AccountConnectionType, AccountSyncStatus } from '@prisma/client'
```

File: apps/workers/src/app/__tests__/queue.integration.spec.ts
Comment (MichaelDeBoey) [inline apps/workers/src/app/__tests__/queue.integration.spec.ts:5]:
```suggestion
import type { AccountConnection, User } from '@prisma/client'
import { AccountConnectionType } from '@prisma/client'
```

File: apps/server/src/app/__tests__/utils/account.ts
Comment (MichaelDeBoey) [inline apps/server/src/app/__tests__/utils/account.ts:38]: I guess we can just keep this as it was since it's the only change in this file?

```suggestion
    const [_deleted, ...securities] = await prisma.$transaction([
```

**Review by MichaelDeBoey — CHANGES_REQUESTED** [review]

File: apps/server/src/app/__tests__/utils/account.ts
Comment (tmyracle) [reply apps/server/src/app/__tests__/utils/account.ts:38]: sure just removing an unused value so ide stops complaining, but ill put it back for now and remove in separate PR

File: apps/server/src/app/__tests__/balance-sync.integration.spec.ts
Comment (tmyracle) [reply apps/server/src/app/__tests__/balance-sync.integration.spec.ts]: can't set category until to generated expression is dropped from the db which I think is in your changes. I'll need to go back in and re-do all the stuff I rolled back after your PR is merged

Comment (MichaelDeBoey) [reply apps/server/src/app/__tests__/balance-sync.integration.spec.ts]: Hmmm too bad that's not possible 😓

File: README.md
Comment (MichaelDeBoey) [inline README.md:40]: I guess we can do this in a separate PR?

```suggestion
*You'll need Docker installed to run the app locally.*
```

**Review by MichaelDeBoey — APPROVED** [review]

File: apps/server/src/app/__tests__/balance-sync.integration.spec.ts
Comment (tmyracle) [reply apps/server/src/app/__tests__/balance-sync.integration.spec.ts]: You can make your changes which will break this test temporarily and then we can go back in and rework the test to use category directly rather than relying on the plaidTypes/subTypes. Think that's the only way to do it unless you want to make the test fixes as part of removing all the plaid stuff

Comment (tmyracle) [reply apps/server/src/app/__tests__/balance-sync.integration.spec.ts]: I'll be ready and waiting for the change to land, I've already made the changes before so it should be a quick fix after I merge in your changes.

File: README.md
Comment (tmyracle) [reply README.md:40]: Dumb format on save, I think its fine.

**Review by MichaelDeBoey — APPROVED** [review]

**Pull request merged**
